### PR TITLE
🛑 = exit

### PR DESCRIPTION
**Pull request by jordancluts**

### CODE REVIEW

**Comment by theogf** [comment]
I think you mistakenly deleted the previous contribution

**Comment by jordancluts** [comment]
It looks okay to me? I see the single edit. I mearly had mistyped `const` as `conts` which I fixed with the 2nd commit.

File: src/WatchJuliaBurn.jl [hunk 1]
```diff
@@ -14,7 +13,7 @@ const ✅ = true
 const 👍 = true
 const 👌 = true
 const 👎 = false
-@eval $(Symbol("@🥩_str")) = $(getfield(Main, Symbol("@raw_str")))
```
Comment (theogf) [inline src/WatchJuliaBurn.jl:17]: You remove this line here, I think it's a merge error

**Comment by jordancluts** [comment]
Gonna make 1 fixed pull request with my 2 ideas and hopefully unbreak things. Sorry I don't use Github much so I'm bad at the pull request process.

**Pull request closed without merging**